app/assets/stylesheets/settings/_general.scss

```diff
@@ -48,10 +46,13 @@ ul[role='tablist'] {
   display: none;
 }
 
-// #wizard h1 {
-//   display: none;
-// }
-
 .is-hidden {
   display: none;
 }
+
+hr {
```

Comment:
that is a separator. 

If you take a look at the layout, it is flexible to avoid it collapse on little screens and to separate elements proportionally in bigger ones. 
For this, I think is better to have a different element drawing that separator than integrate it as a border in one of the elements. Of course, there are many different ways to achieve this. 
<hr> is the tag we should use for separate contents, so I thought that was a justify us of it. We can change for a div but I think that is the case you can use it. (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr)